CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8.8)
+cmake_minimum_required (VERSION 3.0)
```

Comment:
Sorry, doing cmake stuff is thankless work. Its hard to know on which systems you break something and usually somebody is unhappy ;)